packages/shared/src/components/PostOptionsMenu.tsx

```diff
@@ -39,14 +39,21 @@ export default function PostOptionsMenu({
   onMessage,
   onRemovePost,
 }: PostOptionsMenuProps): ReactElement {
-  // eslint-disable-next-line @typescript-eslint/no-unused-vars
-  const { feedSettings } = useFeedSettings();
+  useFeedSettings();
   const { user } = useContext(AuthContext);
   const { reportPost, hidePost } = useReportPost();
   const { unfollowSource, blockTag } = useMutateFilters(user);
   const [reportModal, setReportModal] =
     useState<{ index?: number; post?: Post }>();
 
+  const showMessageAndRemovePost = async (
+    message: string,
+    _postIndex: number,
```

Comment:
No need to accept that as a parameter. You can just use the variable directly

Reply:
It's a dynamic one the report looses it's scope so that's why I have a separate one.

Reply:
Makes sense. Sorry for that. I approved the PR